lib/datadog/tracing/span.rb

```diff
@@ -210,6 +214,16 @@ def start_time_nano
       def duration_nano
         (duration * 1e9).to_i
       end
+
+      # https://docs.datadoghq.com/tracing/visualization/#service-entry-span
+      # A span is a service entry span when it is the entrypoint method for a request to a service.
+      # You can visualize this within Datadog APM when the color of the immediate parent on a flame graph is a different
+      # color. Services are also listed on the right when viewing a flame graph.
+      #
+      # @return [Boolean] `true` if the span is a serivce entry span
+      def service_entry?
+        !!@service_entry
```

Comment:
Minor, but I think maybe `@service_entry == true` is more legible? Always coerces a value to `true`/`false`?